api/config/config_stable.json

```diff
@@ -0,0 +1,22 @@
+{
+  "firecloud": {
+    "debugEndpoints": false,
+    "billingAccountId": "014D91-FCB792-33D2C0",
+    "billingProjectPrefix": "aou-rw-stable-",
+    "registeredDomainName": "all-of-us-registered-stable",
+    "enforceRegistered": false
+  },
+  "cdr": {
+    "defaultCdrVersion": "Test Registered CDR"
+  },
+  "googleCloudStorageService": {
+    "credentialsBucketName": "aou-res-workbench-stable-credentials"
```

Comment:
Not saying we have to do this right now, but if we are going to change naming conventions for credentials bucket, project name, etc. we probably want to change test as well to match. Otherwise it becomes hard to remember where to look for things when switching between environments.

Reply:
I completely agree. I don't like "aou-res-workbench" as a namespace, but that was already there. Let's agree on a convention and fix everywhere.